public/app/plugins/datasource/tempo/SearchTraceQLEditor/TraceQLSearch.test.tsx

```diff
@@ -114,25 +114,37 @@ describe('TraceQLSearch', () => {
     });
 
     await user.click(screen.getByText('Select tag'));
-    jest.advanceTimersByTime(1000);
+    await act(async () => {
+      jest.advanceTimersByTime(1000);
+    });
     await user.click(screen.getByText('foo'));
-    jest.advanceTimersByTime(1000);
+    await act(async () => {
+      jest.advanceTimersByTime(1000);
+    });
     await user.click(screen.getAllByText('Select value')[2]);
-    jest.advanceTimersByTime(1000);
+    await act(async () => {
+      jest.advanceTimersByTime(1000);
+    });
     await user.click(screen.getByText('driver'));
-    jest.advanceTimersByTime(1000);
+    await act(async () => {
+      jest.advanceTimersByTime(1000);
+    });
```

Comment:
Could you double check if we can put all of this in one `await act(async () => {` block (so that the code is a little bit clearer and there are less lines to read/process)?

Reply:
I've tried to do this, but then the test will fail as it cannot find an element with text 'foo'. 

As for the linting issue, I've pushed a fix. I'm surprised that no pre-push check ran for that. It would have saved time.